.gitignore

```diff
@@ -23,3 +23,8 @@ docs/build/*
 vendor
 client.pem
 .chef.login
+
+# Ignore Jetbrains config
+.idea
+# Ignore VS Code
+.vscode
```

Comment:
Please, don't pollute project `.gitignore` with this files, and use [global](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) or [project local](https://help.github.com/en/articles/ignoring-files#explicit-repository-excludes) ignore file instead.